checkov/cloudformation/checks/graph_checks/LambdaOpenCorsPolicy.yaml

```
@@ -0,0 +1,40 @@
+metadata:
+  name: "Ensure no open CORS policy"
+  id: "CKV2_AWS_76"
```

```suggestion
  id: "CKV2_AWS_75"
```

Use the same ID if it's the same check across frameworks

---

Got it!